homeassistant/components/elmax/__init__.py

```diff
@@ -0,0 +1,68 @@
+"""The elmax-cloud integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .common import ElmaxCoordinator
+from .const import (
+    CONF_ELMAX_PANEL_ID,
+    CONF_ELMAX_PANEL_PIN,
+    CONF_ELMAX_PASSWORD,
+    CONF_ELMAX_USERNAME,
+    DOMAIN,
+    ELMAX_PLATFORMS,
+    POLLING_SECONDS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry) -> bool:
+    """Set up elmax-cloud from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    # Retrieve username/password from integration configuration
+    username: str = entry.data[CONF_ELMAX_USERNAME]
+    password: str = entry.data[CONF_ELMAX_PASSWORD]
+    panel_id: str = entry.data[CONF_ELMAX_PANEL_ID]
+    panel_pin: str = entry.data[CONF_ELMAX_PANEL_PIN]
+
+    # Create the API client object and attempt a login, so that we immediately know
+    # if there is something wrong with user credentials
+    coordinator = ElmaxCoordinator(
+        hass=hass,
+        logger=_LOGGER,
+        username=username,
+        password=password,
+        panel_id=panel_id,
+        panel_pin=panel_pin,
```

Comment:
```suggestion
        username=entry.data[CONF_ELMAX_USERNAME],
        password=entry.data[CONF_ELMAX_PASSWORD],
        panel_id=entry.data[CONF_ELMAX_PANEL_ID],
        panel_pin=entry.data[CONF_ELMAX_PANEL_PIN],
```

Reply:
Agreed! @epenet already mentioned this, thanks